src/main/kotlin/uk/gov/dluhc/notificationsapi/rest/StatisticsController.kt

```diff
@@ -55,7 +55,11 @@ class StatisticsController(
     ): CommunicationsStatisticsResponseOAVA {
         val notifications = sentNotificationsService.getNotificationsForApplication(
             sourceReference = applicationId,
-            sourceType = if (oavaService == "postal") SourceType.POSTAL else SourceType.PROXY,
+            sourceType = when (oavaService) {
+                "postal" -> SourceType.POSTAL
+                "proxy" -> SourceType.PROXY
+                else -> SourceType.OVERSEAS
+            },
```

Comment:
I'd be tempted to make overseas explicitly match "overseas", and fall back to a 4xx error - maybe 404. 

e.g.
```kotlin
            sourceType = when (oavaService) {
                "postal" -> SourceType.POSTAL
                "proxy" -> SourceType.PROXY
                "overseas" -> SourceType.OVERSEAS
                else -> throw ResponseStatusException(HttpStatus.NOT_FOUND, "Service $oavaService Not found")
            },
```